src/network.rs

```diff
@@ -83,64 +489,100 @@ impl From<BtcDaiOrder> for PublishOrder {
     }
 }
 
-#[cfg(test)]
-impl Default for Order {
-    fn default() -> Self {
-        Self {
-            inner: BtcDaiOrder::default(),
-            taker: Taker::default(),
+// TODO: The comit::network::orderbook expects a token contract but we
+// assume DAI. Maybe we should use a type that combines dai::Amount
```

Comment:
We should do whatever solution is best for us to handle several dai networks and their contract address. Please do not consider other ERC20 tokens when designing this.

Reply:
Good point.